src/components/Provider/Provider.tsx

```diff
@@ -23,7 +23,14 @@ export interface ProviderProps {
 /**
  * The Provider passes the CSS in JS renderer and theme to your components.
  */
-class Provider extends React.Component<ProviderProps, any> {
+class Provider extends React.Component<ProviderProps> {
+  staticStylesRendered: boolean
+
+  constructor(props: ProviderProps) {
+    super(props)
+    this.staticStylesRendered = false
+  }
+
```

Comment:
```suggestion
staticStylesRendered: boolean = false
```

You can set a value without `constructor`, or I am missing something?

Reply:
Yep, I will remove the constructor. I was trying different things, it is just leftover.